packages/backend/app/Services/Telegram.ts

```diff
@@ -1,53 +1,86 @@
-import { ItelegramConfiguration } from './../../common/interfaces/Itelegram';
+import { IPositiveResponse, INegativeResponse } from './../../common/interfaces/IResponse';
+import { ITelegram } from "../../common/interfaces/Itelegram";
 import axios from "axios";
 import telegramConfiguration from "Config/telegram";
 import { DateTime } from "luxon";
+import { creatingErrMsg, creatingOkMsg } from "../../common/helpers/creatingResponse";
 
-
-class TelegramBot {
-  private readonly config: ItelegramConfiguration;
+export default class TelegramBot {
+  private readonly config: ITelegram;
   constructor() {
     this.config = telegramConfiguration;
   }
 
-  private async sendRequest<T extends Record<string, void>>(
+  protected formatDate = (date: DateTime): string => {
+    return date
+      .setLocale("ru")
+      .setZone("Europe/Moscow")
+      .toLocaleString(DateTime.DATETIME_MED);
+  };
+
+  private async sendRequest(
     path: string,
     _data?: Record<string, void>
-  ): Promise<{ ok: boolean } & Partial<T>> {
+  ): Promise<INegativeResponse | IPositiveResponse<Record<string, void>>> {
     const response = await axios.get(
       `https://api.telegram.org/bot${this.config.telegram_bot_token}${path}`
     );
     return response.data;
   }
 
-  public async sendMsgToTelegram(msg: string): Promise<boolean> {
-    return (
-      await this.sendRequest(
-        `/sendMessage?&chat_id=${this.config.telegram_chat_id}&parse_mode=html&text=${msg}`
-      )
-    ).ok;
+  public async sendMsgToTelegram(
+    id: number,
+    apartmentAddress: string,
+    dateFrom: DateTime,
+    dateTo: DateTime,
+    fixedTotalPrice: number,
+    name: string,
+    phone: string,
+    payment_url: string
+  ): Promise<any> {
+
+    const msg = this.createMessageBody(
+      id,
+      apartmentAddress,
+      dateFrom,
+      dateTo,
+      fixedTotalPrice,
+      name,
+      phone,
+      payment_url
+    );
+
+    try {
+      const telegramResponse = await this.sendRequest(
+        `/sendMessage?&chat_id=1${this.config.telegram_chat_id}&parse_mode=html&text=${msg}`
+      );
+      return creatingOkMsg(telegramResponse)
+    } catch (err) {
+      const error = err as Error
```

Comment:
```suggestion
      if (error instanceof Error) {
            const error = err as Error
      }
```

Ты где-то логируешь ошибку, если не удалось забронировать или не удалось отправить уведомление в ТГ? Надо.